check_borg

```diff
@@ -1,37 +1,41 @@
 #!/bin/sh
-set -o errexit
+
 set -o nounset
 
+# export borg env vars here
+# do not forget to run chmod (e.g. chmod 700 path/to/check_borg)
+export BORG_PASSPHRASE=''
+export BORG_REPO=''
```

Comment:
Works great for me. Both environmental variables can easily be overwritten, if you do so in the right places. Instead of using another config file, I set `BORG_PASSPHRASE` and `BORG_REPO` right there in `check_borg` - one potential culprit less IMHO.